src/components/PeopleTable/PeopleTableRow.jsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Tr,
+  Td,
+  VStack,
+  HStack,
+  Flex,
+  Avatar,
+  Badge,
+  Text,
+  IconButton,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+} from '@chakra-ui/react';
+import { BsThreeDotsVertical } from 'react-icons/bs';
+
+const editAccountInfo = data => {
+  // eslint-disable-next-line no-console
+  console.log(`Editing account info ${data}`);
+};
+
+const editSegmentAssignment = data => {
+  // eslint-disable-next-line no-console
+  console.log(`Editing segment assignment ${data}`);
+};
+
+const clearSegmentAssignment = data => {
+  // eslint-disable-next-line no-console
+  console.log(`Clearing segment assignment ${data}`);
+};
+
+const deletePendingAccount = data => {
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'deletePendingAccount' is assigned a value but never used.